database/migrations/postgres/000001_create_blocks.up.sql

```diff
@@ -1,6 +1,6 @@
 CREATE TABLE blocks (
                         inserted_at TIMESTAMP WITH TIME ZONE  DEFAULT CURRENT_TIMESTAMP NOT NULL,
-                        id BIGINT NOT NULL,
+                        id SERIAL PRIMARY KEY,
```

Comment:
I think the migrations should reflect what we have here i.e. the `id           BIGSERIAL PRIMARY KEY`
https://github.com/bitcoin-sv/arc/blob/31aa8190dff210b87c12378adb7a67cc286aeb4e/blocktx/store/sql/sql.go#L173

Which is the schema of the postgres production db

Reply:
changed to bigserial.